src/main/java/com/google/cloud/tools/appengine/cloudsdk/serialization/CloudSdkVersion.java

```diff
@@ -164,24 +178,12 @@ public boolean equals(Object obj) {
     CloudSdkVersion otherVersion = (CloudSdkVersion) obj;
 
     return Objects.equals(majorVersion, otherVersion.majorVersion)
```

Comment:
One advantage of using HEAD = { -1, -1, -1 } is that we can't create a version from "-1.-1.-1" whereas we could from "999999.999999.999999".

Reply:
Yup.